skeleton/.buildkite/pipeline.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+  - label: Check Hydra evaluation of release.nix
+    commands:
+      - "cd skeleton"  # TODO: remove this line
```

Comment:
Remove this line or change to a different name?

Reply:
I was used to:

```yaml
      - "rm -rf /build/decentralized-software-updates"
      - "cp -R . /build/decentralized-software-updates"
      - "cd /build/decentralized-software-updates"
```

Isn't this needed anymore?

Reply:
Copying the project to `/build` was only required for `.stack-work` caching.
Unfortunately, at present, we don't have any intermediate build product caching at all because the `cache-s3` tool is too slow on packet.net machines.